doc/manuals/devel/subscriptionCache.md

```diff
@@ -182,7 +185,7 @@ typedef struct CachedSubSaved
 
 After having saved that important information in a vector, the entire subscription cache is wiped out and populated from the database, by calling `subCacheRefresh()`.
 
-After repopulation of the subscription cache, the saved information in the `CachedSubSaved` vector is merged into the subscription cache and finally, the `CachedSubSaved` vector is merged into the database, using the function `mongoSubCountersUpdate`, see [special subscription fields](#special-subscription-fields).  
+After repopulation of the subscription cache, the saved information in the `CachedSubSaved` vector is merged into the subscription cache and finally, the `CachedSubSaved` vector is merged into the database, using the function `mongoSubUpdateOnCacheSync`, see [special subscription fields](#special-subscription-fields).
```

Comment:
This is a typo fix. Function `mongoSubCountersUpdate` doesn't exist along the code.

NTC (informative)